src/mount/mount.ceph.c

```diff
@@ -493,9 +510,9 @@ int main(int argc, char *argv[])
 	/* Ensure the ceph key_type is available */
 	modprobe();
 
-	retval = finalize_options(&cmi);
+	retval = append_secret_option(&cmi);
```

Comment:
Nit: perhaps append_key_or_secret or just leave it as is (i.e. finalize_options, which is fairly non-descriptive and doesn't make one think that it's just about "secret")?  One of the reasons behind this regression is confusing naming...